src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -197,6 +205,7 @@ internal sealed override ImmutableArray<string> NotNullWhenFalseMembers
         {
             _property = property;
             _isAutoPropertyAccessor = isAutoPropertyAccessor;
+            _containsFieldKeyword = NodeContainsFieldKeyword(syntax);
```

Comment:
>NodeContainsFieldKeyword(syntax); [](http://example.com/codeflow?start=36&length=33)

Several comments:
- Do we support field keywords in indexers?
- Probably should look specifically in bodies rather than in entire accessor declaration.
- Body vs. ExpressionBody

Fine to add a prototype comment.

Reply:
@AlekseyTs 

- No, we don't support indexers. `CreateBackingField` is only called if `IsIndexer` is `false`. So a change in this area is unlikely to have a behavioral change. But I see there is no need to calculate the field for this case. Is this an unnecessary work you want to eliminate?
- The reason I did that is for when we don't have any accessor declarations at all, e.g `public int P => field;`. But will slightly modify the code to look at `Body` and `ExpressionBody` if we have them.

Reply:
>>Do we support field keywords in indexers?

> Is this an unnecessary work you want to eliminate?

Not only that. In my opinion, it would be simply wrong to set ```_containsFieldKeyword``` to true for an indexer access. Regardless of what we have in its body. According to the specification, there could be no ```field``` keywords there.

> The reason I did that is for when we don't have any accessor declarations at all, e.g ```public int P => field;```.

Do we actually come through this code path for this scenario? I thought the previous constructor is supposed to handle that scenario.